src/BodyPipe.cc

```diff
@@ -133,12 +133,12 @@ BodyPipe::BodyPipe(Producer *aProducer): theBodySize(-1),
     // TODO: teach MemBuf to start with zero minSize
     // TODO: limit maxSize by theBodySize, when known?
     theBuf.init(2*1024, MaxCapacity);
-    debugs(91,7, HERE << "created BodyPipe" << status());
+    debugs(91,7, "created BodyPipe" << status());
```

Comment:
The script did not fix these and similar lines but it can. I left them as is because I was worried that the surrounding lines (without HERE) may use the same bad spacing, and fixing spacing for only some of the lines would increase code inconsistencies. Here is an example:

```
    debugs(91,7, this << " will not consume from " << p);
...
    debugs(91,7, HERE << "created BodyPipe" << status());
```

If you are sure that introducing more inconsistencies is better than preserving bad spacing is a good idea, then please let me know, and I will adjust the script to fix spacing between (most) debugs() section and level parameters.

Alternatively, we can automatically fix spacing in most debugs() lines, with HERE and without. Would you prefer that instead?

In all of the above, I say "most" because whitespace in some exceptionally complicated debugs() expressions may not be fixed. That is OK though. The automation can cover the vast majority of cases, and that is sufficient IMO.